src/ai/susi/server/api/susi/UserService.java

```diff
@@ -19,8 +19,9 @@
 
 package ai.susi.server.api.susi;
 
-import java.util.List;
+import java.util.*;
```

Comment:
don't do that. There is no advantage. This is a no-sense-refactoring. I like to have the exact list of all classes which are used.